### PR TITLE
add disqualified state

### PR DESCRIPTION
**Pull request by dreamfall**

### CODE REVIEW

File: config/initializers/sidekiq_cron.rb [hunk 1]
```diff
@@ -2,6 +2,10 @@
   "email_notification_service" => {
     "cron" => "0 * * * *",
     "class" => "Scheduled::EmailNotificationServiceWorker"
+  },
+  "trigger_audit_deadlines" => {
+    "cron" => "0 * * * *",
+    "class" => "Scheduled::AuditDeadlineWorker"
```
Comment (rusllonrails) [inline config/initializers/sidekiq_cron.rb]: @dreamfall not sure, why we need to run such worker every hour?

File: app/models/form_answer_state_machine.rb [hunk 2]
```diff
@@ -55,6 +57,16 @@ def self.trigger_deadlines
     end
   end
 
+  def self.trigger_audit_deadlines
+    if Settings.after_current_audit_certificates_deadline?
+      current_year = Settings.current.award_year
+
+      current_year.form_answers.where(state: "assessment_in_progress").find_each do |fa|
+        fa.state_machine.perform_transition("disqualified")
```
Comment (rusllonrails) [inline app/models/form_answer_state_machine.rb]: @dreamfall I assume that there is missing condition check on Audit Certificate pretense.

```
if !fa.audit_certificate.blank? || (fa.audit_certificate.present? && fa.audit_certificate.attachment.blank?)
   fa.state_machine.perform_transition("disqualified")
end
```

File: config/initializers/sidekiq_cron.rb (hunk 1 above)
Comment (rusllonrails) [inline config/initializers/sidekiq_cron.rb]: @dreamfall I assume that it's enough that you trigger it in app/services/update_deadline_service.rb

Comment (dreamfall) [inline config/initializers/sidekiq_cron.rb]: @rusllonrails to change the state deadline comes.

**Comment by rusllonrails** [comment]
:+1: Good

**Pull request merged**